packages/app/src/components/PageRenameModal.tsx

```diff
@@ -347,6 +348,7 @@ const PageRenameModal = (): JSX.Element => {
           className="btn btn-primary"
           onClick={rename}
           disabled={submitButtonDisabled}
+          data-testid="rename-page-button"
```

Comment:
Ignore and Resolve this comment if you are still working on this PR.

If not, `data-testid="new-page-name-input"` and `data-testid="rename-page-button"` are not used

Reply:
You are not using `new-page-name-input` at line 255

Please review your own PR before you resolve the commnet.

Reply:
Just removed data-testid in PageRenameModal